source/change-streams/tests/README.rst

```diff
@@ -158,4 +158,7 @@ The following tests have not yet been automated, but MUST still be tested
 #. The ``killCursors`` command sent during the "Resume Process" must not be allowed to throw an exception.
 #. ``$changeStream`` stage for ``ChangeStream`` against a server ``>=4.0`` that has not received any results yet MUST include a ``startAtOperationTime`` option when resuming a changestream.
 #. ``ChangeStream`` will resume after a ``killCursors`` command is issued for its child cursor.
-
+#. ``ChangeStream`` against a server ``>=4.2`` must track the latest ``postBatchResumeToken`` included in an ``aggregate`` or ``getMore`` response if available.
+#. ``ChangeStream`` against a server ``>=4.2`` must resume with the ``resumeAfter`` set to ``_id`` of the most recent document returned when the cursor is in the middle of a batch.
+#. ``ChangeStream`` against a server ``>=4.2`` must resume ``resumeAfter`` set to the ``postBatchResumeToken`` after returning all documents in a batch.
+#. ``ChangeStream`` against a server ``>=4.2`` must resume ``resumeAfter`` set to the ``postBatchResumeToken`` after returning an empty batch.
```

Comment:
These last two tests look OK, although I imagine a complete test of the resume token accessor might also accomplish this provided that drivers are also testing that the accessor's return value always agrees with what would be specified in the `resumeAfter` option for a resuming `aggregate` command's `$changeStream` stage.

Reply:
Done